src/polar/Pie.js

```diff
@@ -74,6 +74,7 @@ class Pie extends Component {
       'spring',
       'linear',
     ]),
+    id: PropTypes.string,
```

Comment:
@Tennen You should add a default value of props `id`, and remove `id = uniqueId('recharts-pie-');`.

Reply:
@xile611 done~